testing/matrix/linter/rules/rules.go

```diff
@@ -564,6 +564,31 @@ func objectSecurityContext(object storage.StoreObject) errors.LintRuleError {
 	return errors.EmptyRuleError
 }
 
+func skipHostNetworkPorts(o *storage.StoreObject, c *v1.Container, p *v1.ContainerPort, hostNetworkUsed bool) bool {
+	if o.Unstructured.GetKind() == "DaemonSet" && strings.HasPrefix(o.Unstructured.GetName(), "chrony") &&
+		o.Unstructured.GetNamespace() == "d8-chrony" && c.Name == "chrony" {
+		if (hostNetworkUsed && p.ContainerPort == 123) || p.HostPort == 123 {
+			return true
+		}
+	}
+
+	if o.Unstructured.GetKind() == "StatefulSet" && o.Unstructured.GetName() == "openvpn" &&
+		o.Unstructured.GetNamespace() == "d8-openvpn" && c.Name == "openvpn-tcp" {
+		if (hostNetworkUsed && p.ContainerPort == 5416) || p.HostPort == 5416 {
+			return true
+		}
+	}
+
+	if o.Unstructured.GetKind() == "DaemonSet" && o.Unstructured.GetName() == "node-local-dns" &&
+		o.Unstructured.GetNamespace() == "kube-system" && c.Name == "coredns" {
+		if (hostNetworkUsed && p.ContainerPort == 53) || p.HostPort == 53 {
+			return true
+		}
+	}
+
+	return false
```

Comment:
Do we need to skip this controller? If so, it must be explained in the code via comments.

Reply:
NTP port was moved to the range and other rules were commented out.